dags/inference/jetstream_pytorch_inference.py

```diff
@@ -42,12 +44,14 @@
           "size": "7b",
           "sleep_time": 120,
           "tpu_version_cores": [(TpuVersion.V5E, 8)],
-          "checkpoint": "gs://inference-benchmarks/models/llama2-7b/pytorch/llama-2-7b-merged",
+          "checkpoint": "gs://inference-benchmarks/models/llama2-7b-chat/pytorch/llama-2-7b-chat-merged",
+          "checkpoint_quantized": "gs://inference-benchmarks/models/llama2-7b-chat/pytorch/llama-2-7b-chat-merged-int8-per-channel",
           "dataset": "openorca",
           "tokenizer": "tokenizer.llama2",
           "batch_sizes": [96],
```

Comment:
It might be useful to use a higher batch size for the quantized model. But in order to do this, we may need to create a separate entry in the `test_models` dictionary. i.e. 
```
"llama3-8b": {quantized: False, batch_sizes=[1, 32, 96]},
"llama3-8b-int8": {quantized: True, batch_sizes=[1, 96, 128, 160]},
```
though this may require a bit more parsing during the nested loop to extract the "llama3-8b" string from the `model` variable.

Reply:
Currently we use 96 for both quantized and un-quantized models.